test-utils/state-viewer/src/epoch_info.rs

```diff
@@ -0,0 +1,213 @@
+use borsh::BorshDeserialize;
+use core::ops::Range;
+use near_chain::{ChainStore, ChainStoreAccess, RuntimeAdapter};
+use near_epoch_manager::EpochManager;
+use near_primitives::account::id::AccountId;
+use near_primitives::epoch_manager::epoch_info::EpochInfo;
+use near_primitives::epoch_manager::AGGREGATOR_KEY;
+use near_primitives::hash::CryptoHash;
+use near_primitives::types::{BlockHeight, EpochHeight, EpochId, ProtocolVersion, ShardId};
+use near_store::{DBCol, Store};
+use std::sync::Arc;
+
+pub(crate) fn print_epoch_info(
+    epoch_id: Option<EpochId>,
+    epoch_height: Option<EpochHeight>,
+    block_hash: Option<CryptoHash>,
+    block_height: Option<BlockHeight>,
+    protocol_version: Option<ProtocolVersion>,
+    validator_account_id: Option<AccountId>,
```

Comment:
maybe, should be enum, when I read `get_epoch_ids`, I realized, it only makes sense for one of those options to be specified.